libs/shared/connected/src/lib/SignatureLists/SignatureLists.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Text, Stack, ActionCard } from '@island.is/island-ui/core'
+import format from 'date-fns/format'
+import { FC } from 'react'
+import {
+  ConnectedComponent,
+  SignatureCollectionList,
+} from '@island.is/api/schema'
+import { useLocalization } from '../../utils'
+import { useGetSignatureLists } from './useGetSignatureLists'
+
+interface SignatureListsProps {
+  slice: ConnectedComponent
+}
+
+const formatDate = (date: string) => {
+  try {
+    return format(new Date(date), 'dd.MM.yyyy')
+  } catch {
+    return date
+  }
+}
+
+export const SignatureLists: FC<
+  React.PropsWithChildren<SignatureListsProps>
+> = ({ slice }) => {
+  const { lists, loading } = useGetSignatureLists()
+  const t = useLocalization(slice.json)
+
+  return (
+    <>
+      {!loading && lists?.length > 0 && (
+        <>
+          <Box marginTop={10} marginBottom={3}>
```

Comment:
That's a lot of margin top, how does this look when you embed it in, I know that some component spacing on the web is a bit weird so I'm just wondering why this number is so high :)

Reply:
haha true, doesnt even need to be there